### ISSUE TITLE
Standalone service fabric cluster - Diagnostic store share

### ISSUE THREAD

**Issue by koderi-dp** [microsoft/service-fabric#527]
Hi,

What is correct way to share diagnostic store folder like in configuration

  "diagnosticsStore": 
        {
            "metadata":  "Please replace the diagnostics file share with an actual file share accessible from all cluster machines. For example, \\\\machine1\\DiagnosticsStore.",
            "dataDeletionAgeInDays": "21",
            "storeType": "FileShare",
            "connectionstring": "\\\\machine1\\DiagnosticsStore"
        },

so the other servers (nodes) can access it.
i have 3 nodes, all in workgroup, now becasue fabric is running as network service, i'm having issues with other 2 nodes, they can't create folder on that share, access denined. a did share it as everyone, full access, tried to add in ntfs premission network service full write, but no luck.

if someone did have same issue (workgroup not domain) please let me know

**Comment by maburlik** [comment]
The standalone package was missing a dll required for creating a standalone cluster with a connection string targeting Azure storage but this has been fixed as of 7.0 CU4. Previously you would have been able to upgrade the configuration to point to Azure storage but initial cluster creation would have been blocked. The latter is possible now with the latest version.

**Comment by strabu** [comment]
Hey @poutler 
I had a similar issue as you are describing here. I'm running Servic Fabric (Windows) on prem Version: 7.1.458.9590.
I wanted to use "storeType": "AzureStorage" instead of "FileShare".
When I copied my working "Primary Connection String" from "Microsoft Azure Storage Explorer" and prefixed it with "xstore:" it did not work.

"Start-ServiceFabricClusterConfigurationUpgrade" returned immediately without throwing an exception or any message. (And yes, I increased clusterConfigurationVersion in the ClusterConfig.json-file, TestConfiguration only said: True, and I could not find traces in the usual places.)

That Version 7.1.458.9590 fails silently when I'm changing the Config is really disappointing and has cost me a whole workday. Such behaviour is mentioned in issues multiple times eg. https://github.com/microsoft/service-fabric-issues/issues/1512 or https://github.com/microsoft/service-fabric-issues/issues/1572 but unfortunately nobody seems to care.

Luckily this was our Dev-Cluster so I removed the whole Cluster and reinstalled it again with my new configuration.
At that point I got an Exception that was telling me that there was an invalid Token in the ConnectionString (not which or what exactly).

After removing ";EndpointSuffix=core.windows.net;" from my ConnectionString I could finally install the cluster and logs were written to Azure storage.
